sed/core/processor.py

```diff
@@ -1112,6 +1194,12 @@ def append_energy_axis(
                 calibration=calibration,
                 **kwds,
             )
+            if self._timed_dataframe is not None:
```

Comment:
Same as for the jittering. Energy axis is electron based and should not be present in the timed_dataframe. I would remove this for energy calibration